x/gov/types/v1beta1/vote.go

```diff
@@ -5,15 +5,8 @@ import (
 	"strings"
 
 	sdkmath "cosmossdk.io/math"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// NewVote creates a new Vote instance.
-func NewVote(proposalID uint64, voter sdk.AccAddress, options WeightedVoteOptions) Vote {
```

Comment:
nit: do we need an API breaking changelog?

Reply:
+1 and this should be reverted in the backport PR if we backport this.